packages/dev/core/src/Behaviors/Meshes/sixDofDragBehavior.ts

```diff
@@ -199,7 +199,7 @@ export class SixDofDragBehavior extends BaseSixDofDragBehavior {
 
         if (pointerCount === 1) {
             this._targetPosition.copyFrom(this._ownerNode.absolutePosition);
-            this._targetOrientation.copyFrom(this._ownerNode.absoluteRotationQuaternion);
+            this._targetOrientation.copyFrom(this._ownerNode.rotationQuaternion);

```

Comment:
Why is position and scaling stay absolute, and only rotation uses local rotation?

Reply:
I think it's the reverse. the PR did change position to absoluteposition to fix a bug and did the replacement for rotation and scale as well to be consistent. But actually, it introduced a regression with rotation. Do you confirm @carolhmj ?

Reply:
Multipointer was also changed to absolute, is it still an issue?

Reply:
Let's wait for @carolhmj 's input on this fix and if it's ok then I'll do and test the multipoint.

Reply:
I changed to absolutePosition/Rotation/Scale because I removed this line https://github.com/BabylonJS/Babylon.js/pull/14708/files#diff-867020a1c839af098f39ec5a7d63e25c7fa37dfd2e507e2143f86bafe643e85aL202 which was using setPosition(null) to make the node's position be the absolute position. If you copy only the local rotation, wouldn't that ignore the parent's rotation?

Reply:
@CedricGuillemet - i'll let you resolve this